polkadot/erasure-coding/src/lib.rs

```diff
@@ -379,6 +472,25 @@ mod tests {
 		assert_eq!(reconstructed, available_data);
 	}
 
+	#[test]
+	fn round_trip_systematic_works() {
```

Comment:
Didn't notice this 👍🏻 However, quickcheck also works like a fuzzer. I like the `Arbitrary` approach of proptesting more, because it's easier to express invariants for the input than with hongfuzz.

Also, the fuzz target is not being run in the CI and also tests fewer cases in its current form. (only uses a static number of validators)